azure/services/managedclusters/spec_test.go

```diff
@@ -110,6 +110,121 @@ func TestParameters(t *testing.T) {
 				g.Expect(gomockinternal.DiffEq(result).Matches(sampleCluster)).To(BeTrue(), cmp.Diff(result, getSampleManagedCluster()))
 			},
 		},
+		{
+			name:     "managedcluster does not exist without DNSServiceIP",
```

Comment:
Can we please update the name to better understand this unit test? 
```suggestion
			name:     "managedcluster without a DNSServiceIP and no pre-existing managedClusterSpec",
```